app/views/spree/shared/_review.html.erb

```diff
@@ -1,29 +1,29 @@
-<div class="review" itemprop="reviews" itemscope itemtype="http://schema.org/Review">
-  <span class="reviews-rating" title="<%= txt_stars(review.rating) %>">
-    <%= render 'spree/reviews/stars', stars: review.rating %>
-  </span>
-  <span itemprop="name"><%= review.title %></span>
-  <br/>
-  <span class="attribution"><%= Spree.t(:submitted_on) %>
-    <strong><%= l review.created_at.to_date %></strong>
-  </span>
-  <span itemprop="datePublished" content="<%= review.created_at.to_date.to_s %>"></span>
-  <span itemprop="reviewRating" content="<%= review.rating %>"></span>
-  <% if review.show_identifier %>
-    <% if Spree::Reviews::Config[:show_email] && review.user %>
-      <span itemprop="author"><%= review.user.email %></span>
-    <% else %>
-      <span itemprop="author"><%= review.name %></span>
-    <% end %>
-  <% else %>
-    <span itemprop="author"><%= Spree.t(:anonymous) %></span>
-  <% end %>
-  <div itemprop="reviewBody">
-    <%= simple_format(review.review) %>
+<% for review in (Spree::Reviews::Config[:track_locale] ? @product.reviews.localized(I18n.locale) : @product.reviews).default_approval_filter.preview %>
+  <div id="review">
+    <table class="table table-striped table-bordered">
+      <tbody>
+        <tr>
+          <td style="width: 50%;"><strong><%= review.name%></strong></td>
+          <td class="text-right"><span class="attribution"><%= Spree.t('Date:') %>
+      <strong><%= review.created_at.to_date %></strong>
+    </span></td>
+        </tr>
+        <tr>
+          <td colspan="2">
+            <p><%= review.review %></p>
+            <div class="ratings">
+              <div class="rating-box">
+                <span class="reviews-rating" title="<%= txt_stars(review.rating) %>">
+                  <%= render 'spree/reviews/stars', stars: review.rating %>
+              </div>
+            </div>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <div class="text-right"></div>
   </div>
-  <% if Spree::Reviews::Config[:feedback_rating] && (!Spree::Reviews::Config[:require_login] || spree_current_user) %>
```

Comment:
Why we remove this section. What is the purpose of this section. Need you explanation on that.

Reply:
its add in partial